components/notifications-service2/integration_test/rules_crud_test.go

```diff
@@ -409,7 +412,7 @@ func TestUpdateRuleValidation(t *testing.T) {
 	// as AddRule does. We don't explicitly test those because it's repetitive
 }
 
-func TestDeleteRule(t *testing.T) {
```

Comment:
Not this line, but just above (L394) there is a typo: "contstraint".

Reply:
I will add this to the next PR I'm working on.